nservicebus/upgrades/5to6/distributor.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Deprecated Distributor in NServiceBus Version 6
+tags:
+ - upgrade
+ - migration
+related:
+ - samples/scaleout/distributor-upgrade
```

Comment:
I'm not too familiar with the sample and whether it needs some more related to text to it. Maybe @SzymonPobiega can chime in on that question?